docs/api-docs/storefront/widgets/widgets-tutorial.md

```diff
@@ -222,51 +220,70 @@ Accept: application/json
 |---|---|
 | `widget_uuid` | UUID of the widget |
 | `entity_id` |  page, category, brand, or product ID |
-| `sort_order` | widget sort order |
-| `region` | region the template will show |
-| `template_file` | template file region added to |
+| `sort_order` | widget placement order |
+| `region` | region to place the widget in |
+| `template_file` | template file to target|
 | `status` | active or inactive |
 
+`entity_id` depends on the type of page; for example, for product pages, it is the product ID, and for category pages, it is the category ID. To place your widget on a specific category page, you need to provide a category ID. To retrieve available category IDs, send a `GET` request to [`/v3/catalog/categories`](https://developer.bigcommerce.com/api-reference/store-management/catalog/category/getcategories). If you omit `entity_id`, the widget will appear on all of the category pages.

```

Comment:
`entity_id` depends on the type of page; for example, for product pages, it is the product ID, and for category pages, it is the category ID. To place your widget on a specific category page, you need to provide a category ID. To retrieve available category IDs, send a `GET` request to [`/v3/catalog/categories`](https://developer.bigcommerce.com/api-reference/store-management/catalog/category/getcategories). If you omit `entity_id`; the widget will appear on all of the category pages.